modules/fedimint-ln-common/src/api.rs

```diff
@@ -22,47 +23,63 @@ use crate::contracts::outgoing::OutgoingContractAccount;
 use crate::contracts::{ContractId, DecryptedPreimageStatus, FundedContract, Preimage};
 use crate::{ContractAccount, LightningGateway, LightningGatewayAnnouncement};
 
+/// Trait defining all JSON-RPC commands that can be made to lightning module
+/// RPC server
 #[apply(async_trait_maybe_send!)]
 pub trait LnFederationApi {
+    /// Fetch consensus block height of lightning module
     async fn fetch_consensus_block_count(&self) -> FederationResult<Option<u64>>;
+    /// Wait for a consensus block height to be reached
+    async fn wait_block_height(&self, block_height: u64) -> FederationResult<()>;
+    /// Fetch a lightning module [`ContractAccount`] by ID
     async fn fetch_contract(
         &self,
         contract: ContractId,
     ) -> FederationResult<Option<ContractAccount>>;
+    /// Await fedration consensus on a lightning module [`ContractAccount`]
     async fn wait_contract(&self, contract: ContractId) -> FederationResult<ContractAccount>;
-    async fn wait_block_height(&self, block_height: u64) -> FederationResult<()>;
+    /// Fetch [`OutgoingContractAccount`] for a given contract ID
+    async fn fetch_outgoing_contract(
+        &self,
+        id: ContractId,
+    ) -> FederationResult<OutgoingContractAccount>;
+    /// Wait for the lightning gateway to successfully user client escrow
+    /// contract for failed outgoing payment
     async fn wait_outgoing_contract_cancelled(
         &self,
         contract: ContractId,
     ) -> FederationResult<ContractAccount>;
+    /// Check whether a threshold-decrypted preimage has been decrypted, is
+    /// being decrypted, or hasn't started decryption yet.
     async fn get_decrypted_preimage_status(
         &self,
         contract: ContractId,
     ) -> FederationResult<(IncomingContractAccount, DecryptedPreimageStatus)>;
+    /// Waits for preimage decryption to complete
     async fn wait_preimage_decrypted(
         &self,
         contract: ContractId,
     ) -> FederationResult<(IncomingContractAccount, Option<Preimage>)>;
+    /// Fetch an [IncomingContractOffer]
```

Comment:
I haven't verified, but I think markdown link check is failing since this doesn't have backticks.